pkg/util/kubernetes/apiserver/apiserver.go

```diff
@@ -9,20 +9,31 @@ package apiserver
 
 import (
 	"context"
+	"encoding/json"
 	"io/ioutil"
 	"time"
 
 	log "github.com/cihub/seelog"
 	"github.com/ericchiang/k8s"
-	yaml "gopkg.in/yaml.v2"
+
+	"github.com/ericchiang/k8s/api/v1"
+
+	"fmt"
 
 	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/tagger/collectors"
 	"github.com/DataDog/datadog-agent/pkg/util/retry"
-	"github.com/ericchiang/k8s/api/v1"
 )
 
 var globalApiClient *APIClient
 
+const (
+	configMapDCAToken = "configmaptokendca"
```

Comment:
it bugs me that words are not in the same order, maybe we should rename `configmaptokendca` to `configmapdcatoken`

Reply:
🐛  accepted